### DISCUSSION TITLE
Selecting rows causes table to scroll up to the first row

### DISCUSSION THREAD

**Discussion by MarcWUW** [TanStack/table#5704] [General]
We are using material react table with select and multi select checkboxes enabled. When we scroll down and select a row, the table scrolls back to the first row. So if we can see 20 rows at a time and scroll down to row 180 and want to select rows 180 and 181, as soon as we select row 180 the table scrolls back to row 1. We then have to scroll back down to select row 181.

**Comment by xiaojieyang** [comment]
I am having the same issue, any update on this ?

**Comment by kartikpersistent** [comment]
I am having the same issue any workaround for this?

**Comment by ErykBeru** [comment]
Same issue (svelte version), any update to tableOptions causes a jump to scrollTop 0
